src/main/java/frc/trigon/robot/poseestimation/poseestimator/StandardDeviations.java

```diff
@@ -0,0 +1,58 @@
+package frc.trigon.robot.poseestimation.poseestimator;
+
+import edu.wpi.first.math.geometry.Transform2d;
+
+/**
+ * Standard Deviations represent how ambiguous an estimated pose is using calibrated gains.
+ * The higher the value is, the more ambiguous the pose is and the less trustworthy the result.
+ * Standard Deviations are used to reduce noise in a pose estimate result by accounting for how much each result is wrong by.
+ */
+public class StandardDeviations {
+    private final double translation, theta;
```

Comment:
The names translation and theta could be misleading. How about translationStanderdDeviations and thetaStandardDeviations? Or translationAmbiguaty etc

Reply:
It's inside the Standard deviations class. That's just redundancy. It can't be anything else

Reply:
I still think it can be misleading